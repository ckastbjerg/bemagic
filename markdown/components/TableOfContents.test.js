/* eslint-env jest */
const TableOfContents = require('./TableOfContents');

test('it works for simple components', () => {
    expect(TableOfContents({
        themes: {},
        modifiers: {},
        elements: {},
        states: {},
    })).toMatchSnapshot();
});

test('it works for complex components', () => {
    expect(TableOfContents({
        themes: { theme: {} },
        modifiers: { modifier: {} },
        elements: { element: {} },
        states: { state: {} },
    })).toMatchSnapshot();
});
