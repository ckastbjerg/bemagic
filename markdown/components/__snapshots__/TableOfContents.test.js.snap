// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works for complex components 1`] = `
"
        ###### Table of contents
        
        
        
        
        - **[Component overview](#component-overview)**
        - **[Contributing](#contributing)**
    "
`;

exports[`it works for simple components 1`] = `
"
        ###### Table of contents
        
        
        
        
        - **[Component overview](#component-overview)**
        - **[Contributing](#contributing)**
    "
`;
